clap_derive/tests/subcommands.rs

```diff
@@ -162,13 +162,14 @@ fn test_tuple_commands() {
 }
 
 #[test]
+#[ignore] // FIXME (@CreepySkeleton)
```

Comment:
This is not working yet. I know what this is all about, I know how to fix it, I'll get to it tomorrow.

Reply:
Upon further review, I'm totally convinced that this is an error in the test, see my comment below.